delegate/delegate/CollectionViewCellDelegate.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  ColllectionViewCellDelegate.swift
+//  delegate
+//
+//  Created by Risako Kato on 2022/08/04.
+//
+
+protocol CollectionViewCellDelegate {
```

Comment:
@risako070310 このprotocolはAnyObjectに準拠してあげると良きです！以前はclassを使用していました。
```suggestion
protocol CollectionViewCellDelegate: AnyObject {
```